tests/backup/backup_psa_test.go

```diff
@@ -898,6 +899,12 @@ var _ = Describe("{RestoreFromHigherPrivilegedNamespaceToLower}", Label(TestCase
 	})
 	JustAfterEach(func() {
 		defer EndPxBackupTorpedoTest(scheduledAppContexts)
+
+		//Resetting the pipeline app list
+		defer func() {
+			Inst().AppList = pipelineAppList
+		}()
+
```

Comment:
The defer should come in `JustBeforeEach` itself. Refer to other tests where we have done this.

Reply:
done